backend/app/Http/Controllers/UserController.php

```diff
@@ -93,6 +94,23 @@ public function destroy($id) {
         );
     }
 
+    public function update_password(Request $request) {
+        $user = User::find(Auth::id());
```

Comment:
```suggestion
        $user = Auth::user();
```

Reply:
same sir, i still need to get the `App\Models\User`

![image](https://user-images.githubusercontent.com/95029803/149902424-c45d3dae-deae-4db0-968f-5513fb0ab13b.png)

Reply:
try auth()->user(), if it works

Reply:
same sir its still Authenticable na class, 
the parent class of User Model is Authenticable 
but i need the User Model itself